docs/clone-sst.md

```diff
@@ -0,0 +1,163 @@
+
+For Percona XtraDB Cluster, variables controlling the State Snapshot Transfer (SST) mechanism, such as `wsrep_sst_allowed_methods` and `wsrep-sst-method`, must be set before server startup.
+
+Here is an example:
+
+```
+[mysqld]
+wsrep_sst_allowed_methods = xtrabackup-v2,clone
+```
+
+On Joiner, you must have the following setting. You should set this variable in the [mysqld] section of the my.cnf configuration file. Here is an example:
+
+```
+[mysqld]
+wsrep_sst_method = clone
+```
+
+To enable SSL, ensure that certificates are manually or automatically placed in a specific location on both the donor and joiner servers. Avoid using the data directory because the clone process modifies this directory.
+
+We recommended that you explicitly set the SSL certificates in the my.cnf as:
+
+```
+[client]
+ ssl-ca = /<path>/ca.pem
+ ssl-cert = /<path>/client-cert.pem
+ ssl-key = /<path>/client-key.pem
+[mysqld]
+ ssl-ca = /<path>/ca.pem
+ ssl-cert = /<path>/server-cert.pem
+ ssl-key = /<path>/server-key.pem
+```
+
+ or you can use the following for clone client (Joiner) side 
+ 
+ ```
+[mysqld]
+ clone_ssl_ca=/path/to/ca.pem
+ clone_ssl_cert=/path/to/client-cert.pem
+ clone_ssl_key=/path/to/client-key.pem
+ ```
+ 
+Where <path> is not the data directory.
+
+## Variables
+
+There are three moments where the Joiner or Donor must wait for the other to do an action. All the listed variables have a configurable timeout:
+
+`joiner-timeout-wait-donor-message=60`
+`donor-timeout-wait-joiner=200`
+`joiner-timeout-clone-instance=90`
+
+
+The `joiner-timeout-wait-donor-message` variable is when the Joiner waits for the Donor to answer, indicating whether the action to perform is SST or IST. The default value is set to 60 seconds, which is normally more than enough for any system. At timeout, the process aborts. 
+
+The `donor-timeout-wait-joiner` variable tells the Donor to wait for the Joiner to set up the initial MYSQL instance; the default is 200 seconds, but this value may need to be increased in very slow systems. In any case, the Donor log will clearly report the countdown and, if it happens, the timeout. 
+
+The `joiner-timeout-clone-instance` variable is the Joiner waiting to correctly see the MySQL instance where the clone action will occur. The default timeout is 90 seconds.
+
+You can set the three variables in my.cnf:
+
+[sst]
+joiner-timeout-wait-donor-message=60
+onor-timeout-wait-joiner=200
+joiner-timeout-clone-instance=90
+
+
+## Debug the process
+
+In the same context, if you must debug the process and need more information, you can enable the debug output in my.cnf:
+
+[sst]
+wsrep-debug=true
+
+Finally, as for Xtrabackup SST, the default port used by the process is the 4444.
```

Comment:
It is configurable via `wsrep_sst_receive_address`

Reply:
As before I think we should have this in the common section, given the usage and behavior is the same.